examples/next-js/pages/[...slug].js

```diff
@@ -8,43 +8,47 @@ import Nav from '../components/nav';
 const BUILDER_API_KEY = 'YOUR_KEY';
 builder.init(BUILDER_API_KEY);
 
-class CatchallPage extends React.Component {
-  static async getInitialProps({ res, req, asPath }) {
-    // Get the upcoming route full location path and set that for Builder.io page targeting
-    const path = asPath.split('?')[0];
+const getServerSideProps = async ({req, res}) => {
+  // Get the upcoming route full location path and set that for Builder.io page targeting
+  const [path] = req.url.split('?');
 
-    // 'page' is the model name for your pages. If you made a new model with a different name,
-    // such as 'my-page', use `builder.get('my-page', ...)
-    const page = await builder
-      .get('page', { req, res, userAttributes: { urlPath: path } })
-      .promise();
+  // 'page' is the model name for your pages. If you made a new model with a different name,
+  // such as 'my-page', use `builder.get('my-page', ...)
+  const page = await builder
+    .get('page', { req, res, userAttributes: { urlPath: path } })
+    .promise();
 
-    if (res && !page) {
-      res.statusCode = 404;
-    }
-    return { builderPage: page };
+  if (res && !page) {
+    res.statusCode = 404;
   }
 
-  render() {
-    const page = this.props.builderPage;
-    return (
-      <>
-        <Nav />
-        <div>
-          {page ? (
-            <>
-              <Head>
-                <title>{page.data.title}</title>
-              </Head>
-              <BuilderComponent model="page" content={page} />
-            </>
-          ) : (
-            <div>Page not found!</div>
-          )}
-        </div>
-      </>
-    );
-  }
-}
+  return {
+    props: {
+      builderPage: {
+        ...page,
+        testVariationName: page?.testVariationName || null
```

Comment:
Needed because if `undefined` it cannot be serialized.

![image](https://user-images.githubusercontent.com/709159/80963934-436c1880-8e10-11ea-87c5-065f78e002fd.png)